### PR TITLE
Remove output logs when upgrading clusters

### PR DESCRIPTION
**Pull request by apoorva11029**
Fix for https://github.com/noironetworks/support/issues/1256

### CODE REVIEW

**Review by snaiksat — CHANGES_REQUESTED** [review]
Looks good once we remove the test script.

**Comment by apoorva11029** [comment]
> Looks good once we remove the test script.

Thanks, removed it.

File: provision/acc_provision/acc_provision.py [hunk 1]
```diff
@@ -1181,14 +1181,15 @@ def generate_kube_yaml(config, operator_output, operator_tar, operator_cr_output
...
-              config["kube_config"]["system_namespace"],
-              ",".join(kube_objects),
-              str(config["registry"]["configuration_version"])))
+        if not config["provision"]["upgrade_cluster"]:
+            info("Delete stale objects from older deployments using:")
+            info("  %s -n %s delete %s -l "
+                 " 'aci-containers-config-version,"
+                 "aci-containers-config-version notin (%s)'" %
+                 (config["kube_config"]["kubectl"],
+                 config["kube_config"]["system_namespace"],
```
Comment (snaiksat) [inline provision/acc_provision/acc_provision.py:1190]: pep-8 doesnt seem to like the indentation, i think one extra space is needed here and remaining lines below (per the original formatting in the older version).

Comment (apoorva11029) [reply provision/acc_provision/acc_provision.py]: Yes, needed to add an extra space. Added now, please check!

**Review by snaiksat — APPROVED** [review]

**Pull request merged**